### DISCUSSION TITLE
issubclass() arg 1 must be a class

### DISCUSSION THREAD

**Discussion by Rmaram07** [eyurtsev/kor#193] [General]
After the recent code push, I get this error "TypeError: issubclass() arg 1 must be a class". This might be something related to chain.run() which is replaced by chain.predict_and_parse(). Can anyone help me fix this issue?

**Comment by eyurtsev** [comment]
Hi @Rmaram07,

Could you open an issue rather than a discussion for this topic and include a stack trace and a minimal code example to reproduce?

**Comment by eyurtsev** [comment]
Relocated to an issue

https://github.com/eyurtsev/kor/issues/195
